.github/workflows/lint-prettier.yml

```diff
@@ -0,0 +1,14 @@
+name: ESLint and Prettier
+on: [push]
+jobs:
+  eslint-prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - run: npm install
+      - name: Running ESLint
+        run: npm run lint
+      - name: Running Prettier
+        run: npm run check-format
+      - run: echo "The commit passed all formatting checks! 🚀 🚀 💯 💯 🏅 🏅 🤓 🤓 🤓 🤓 🤓"
```

Comment:
The .yml file sets up the Github Actions workflow. 
[Github Actions ](https://docs.github.com/en/actions/quickstart)will run them automatically.